ansible/configs/ocp4-disconnected-osp-lab/files/clouds.yaml.j2

```diff
@@ -2,11 +2,16 @@ clouds:
   {{ osp_project_name }}:
     auth:
       auth_url: "{{ osp_auth_url }}"
+{% if osp_project_create | bool %}
```

Comment:
I don't think I like this. We should just use `osp_auth_username_member` going forward. Even if we are creating a project, we set that var at the beginning of the role so it is always available. There is no reason here to define the user as `guid-user` again.

Same for the password...`osp_auth_password_member` should be fine whether the project is being created or not.

Reply:
I wanted to stay backward compatible, but I'm fine with that :)